### ISSUE TITLE
Popover color accessibility

### ISSUE THREAD

**Issue by daviddkkim** [influxdata/clockface#682]
Look into a solid background or darker glassmorphic background for popover

![image (11)](https://user-images.githubusercontent.com/12561526/142048937-27d2a451-13f9-4dbd-bf68-858be6635b7d.png)

When we have light background, the current preset doesn't work
